Docs/Book/Cookbook.md

```diff
@@ -267,7 +267,7 @@ Examples of using it:
     rms = rdMolAlign.AlignMol(mol1, mol2)
     tani = rdShapeHelpers.ShapeTanimotoDist(mol1, mol2)
     prtr = rdShapeHelpers.ShapeProtrudeDist(mol1, mol2)
-    print rms, tani, prtr
+    print('{0:.11f} {1:.11f} {2:.13f}'.format(rms, tani, prtr))
```

Comment:
This looks overkill, but will reproduce the expected result. We could change this to a more reasonable format statement.